hebi_a-2085-04-parallel-gripper_moveit_config/package.xml

```diff
@@ -3,7 +3,7 @@
   <name>hebi_a-2085-04-parallel-gripper_moveit_config</name>
   <version>0.3.0</version>
   <description>
-     An automatically generated package with all the configuration and launch files for using the A-2085-04 with the MoveIt! Motion Planning Framework
+     An automatically generated package with all the configuration and launch files for using the A-2085-04-parallel-gripper with the MoveIt! Motion Planning Framework
```

Comment:
Tiny thing -- this name change isn't made in the `hebi_a-2084-01-parallel-gripper_moveit_config`.  I'm not sure which version I like better -- just keeping the name as `A-2085-04`/etc, or explicitly noting the arm_config value here.